src/Roslyn.Suppressions/Roslyn.Suppressions/SupportedSuppressionBuilder.g.cs

```diff
@@ -48,7 +48,7 @@ private static ImmutableArray<SuppressionDescriptor> GetDescriptors()
                 // If the number of diagnostic ids changes significantly or decreases when the analysers are updated,
                 // investigate! There may be a problem with code generator.
                 // *************************************************************************************************
-                // Number of unique diagnostic ids (C# and VB.NET): 429
+                // Number of unique diagnostic ids (C# and VB.NET): 435
```

Comment:
Whenever we update the Sonar Roslyn analyzers, we should re-run the `.tt` file that generates the list of issues that can be suppressed.

I've re-run it manually this time, and added #3335 to the backlog.